logstreams/src/main/java/io/zeebe/distributedlog/restore/snapshot/impl/InvalidSnapshotReplicationResponse.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.distributedlog.restore.snapshot.impl;
+
+import io.atomix.cluster.MemberId;
+import io.zeebe.distributedlog.restore.snapshot.SnapshotRestoreRequest;
+
+public class InvalidSnapshotReplicationResponse extends RuntimeException {
+
```

Comment:
Wow ok didnt expected that :see_no_evil: ` extends RuntimeException {`
If you extend an Exception please add as suffix `*Exception` to your class name.
This is a Java convention.

Reply:
Do you really need a custom exception type? This only class only build the exception message, the getters are not used is this correct?

Reply:
Didn't pay attention to the name :-). You are right. Custom exception is not adding any value here. May be a RuntimeException with a clear message will be enough here.